com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -17,6 +17,31 @@ public sealed class NetworkObject : MonoBehaviour
         [SerializeField]
         internal uint GlobalObjectIdHash;
 
+        /// <summary>
+        /// Gets the Prefab Hash Id of this object if the object is registerd as a prefab otherwise it returns 0
+        /// </summary>
+        [HideInInspector]
+        public uint PrefabHashId
```

Comment:
```suggestion
        public uint PrefabIdHash
```

it's actually a hash of an ID, not an ID of a hash :)

Reply:
😁 Oh Yeah!